ecommerce/utils.py

```diff
@@ -103,3 +104,16 @@ def make_checkout_url(*, product_id=None, code=None):
     if code is not None:
         query_params["code"] = code
     return f"{base_checkout_url}?{urlencode(query_params)}"
+
+
+def get_product_from_querystring_id(model, qs_product_id):
```

Comment:
Please get rid of the `model` argument here, just use `Product` in the body of the function, and change the calls to this function to drop the `models.Product` argument.

Reply:
We can't directly import `Product` model in `utils` because we are already importing it in `ecommerce/serializers` and `ecommerce/views` which raises circular dependency leading to error while importing `Product` model, so that's why I had chosen that way of passing the model.

Reply:
Then we should move this to ecommerce/api.py, where it probably belongs anyway

Reply:
ok